usaspending_api/agency/v2/views/obligations_by_award_category.py

```diff
@@ -69,8 +69,8 @@ def format_results(self, results):
                 results_map[key] = round(obligation_value, 2)
             else:
                 other_total += obligation_value
+        results_map["other"] += round(other_total)
```

Comment:
Could we just remove this and the else above? Cannot remember if this was put in place to cover a specific reason or if the `else` is redundant.

Reply:
i'm pretty sure the idea was that if there was an award with a category that we don't have in the mapping it would go in that "other" sum, but I don't know if we have any data like that.

Reply:
Makes sense. Code LGTM then, but Travis has a few complaints.